include/Server.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef SERVER_HPP
+# define SERVER_HPP
+# include <vector>
+# include <iostream>
+# include <sys/socket.h>
+# include <sys/types.h>
+# include <sys/event.h>
+# include <sys/time.h>
+# include <stdlib.h>
+# include <unistd.h>
+# include <netdb.h>
+# include <signal.h>
+# include "macro.hpp"
```

Comment:
I don't see the header used in the class.

Reply:
Removed and pushed!